<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="25" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
11:02:11 INFO  Dir:   plugin_apama/
11:02:11 INFO  Title: Validate apama plugin install
11:02:11 INFO  ==============================================================
11:02:11 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.08 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
11:02:11 INFO  Dir:   plugin_apama/
11:02:11 INFO  Title: Validate mon file support of apama plugin
11:02:11 INFO  ==============================================================
11:02:11 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.03 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
11:02:11 INFO  Dir:   plugin_apama/
11:02:11 INFO  Title: Validate apama plugin remove
11:02:11 INFO  ==============================================================
11:02:11 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.02 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
11:02:11 INFO  Dir:   plugin_apama/
11:02:11 INFO  Title: Validate apama plugin updating one project with another
11:02:11 INFO  ==============================================================
11:02:11 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.02 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>11:52:43 INFO  ==============================================================
11:52:43 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
11:52:43 INFO  Dir:   plugin_apama/
11:52:43 INFO  Title: Validate apama plugin updating one project with another
11:52:43 INFO  ==============================================================
11:52:43 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
11:52:43 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
11:52:44 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
11:52:44 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
11:52:45 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
11:52:46 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
11:52:46 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
11:52:47 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
11:52:47 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
11:52:47 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:52:47 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

11:52:48 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
11:52:48 WARN  Aborted test due to BLOCKED outcome
11:52:48 INFO  
11:52:48 INFO  cleanup:
11:52:48 INFO  Removing apama project
11:53:03 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
11:53:04 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
11:53:04 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
11:53:04 INFO  
11:53:04 INFO  Test duration: 21.64 secs
11:53:04 INFO  Test final outcome:  BLOCKED
11:53:04 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
11:53:04 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>11:56:02 INFO  ==============================================================
11:56:02 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
11:56:02 INFO  Dir:   plugin_apama/
11:56:02 INFO  Title: Validate apama plugin updating one project with another
11:56:02 INFO  ==============================================================
11:56:02 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
11:56:03 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
11:56:04 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
11:56:04 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
11:56:05 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
11:56:06 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
11:56:06 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
11:56:07 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
11:56:07 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
11:56:07 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:56:07 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

11:56:07 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
11:56:07 WARN  Aborted test due to BLOCKED outcome
11:56:07 INFO  
11:56:07 INFO  cleanup:
11:56:07 INFO  Removing apama project
11:56:23 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
11:56:23 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
11:56:24 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
11:56:24 INFO  
11:56:24 INFO  Test duration: 21.44 secs
11:56:24 INFO  Test final outcome:  BLOCKED
11:56:24 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
11:56:24 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.06 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>11:34:45 INFO  ==============================================================
11:34:45 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:34:45 INFO  Dir:   software_management_end_to_end/
11:34:45 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:34:45 INFO  ==============================================================
11:34:46 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:34:46 INFO  EnvironmentC8y Setup
11:34:46 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:35:01 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:35:03 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:35:04 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:35:04 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:35:07 INFO  State of current operation: SUCCESSFUL
11:35:07 INFO  Assert that (False == value) with value=False ... passed
11:35:07 INFO  Executed sudo, exit status 0
11:35:08 INFO  Executed sudo, exit status 0
11:35:09 INFO  Executed sudo, exit status 0
11:35:13 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:35:13 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:35:13 INFO  Delaying execution by 8 seconds
11:35:34 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:35:35 INFO  Response status: 201
11:35:35 INFO  Response to action: {
    "creationTime": "2022-04-28T11:35:35.178Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17899079",
    "id": "17899079",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
11:35:35 INFO  Started operation: {'creationTime': '2022-04-28T11:35:35.178Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17899079', 'id': '17899079', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
11:35:35 WARN  , retrying in 0.5 seconds...
11:35:35 WARN  , retrying in 0.5 seconds...
11:35:36 WARN  , retrying in 0.5 seconds...
11:35:36 WARN  , retrying in 0.5 seconds...
11:35:37 WARN  , retrying in 0.5 seconds...
11:35:37 WARN  , retrying in 0.5 seconds...
11:35:38 WARN  , retrying in 0.5 seconds...
11:35:38 WARN  , retrying in 0.5 seconds...
11:35:39 WARN  , retrying in 0.5 seconds...
11:35:39 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
11:35:40 WARN  AssertionError: ... blocked 
11:35:40 INFO  
11:35:40 INFO  cleanup:
11:35:41 INFO  Executed sudo, exit status 0
11:35:41 INFO  Executed sudo, exit status 0
11:35:45 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:35:46 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:35:46 INFO  Delaying execution by 8 seconds
11:36:08 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:36:12 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:36:12 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:36:12 INFO  
11:36:12 INFO  Test duration: 86.88 secs
11:36:12 INFO  Test final outcome:  BLOCKED
11:36:12 INFO  Test outcome reason: AssertionError:
11:36:12 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>11:56:24 INFO  ==============================================================
11:56:24 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
11:56:24 INFO  Dir:   software_management_end_to_end/
11:56:24 INFO  Title: Install package to directory specified in tedge config set tmp.path
11:56:24 INFO  ==============================================================
11:56:25 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:56:25 INFO  EnvironmentC8y Setup
11:56:26 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:56:40 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:56:41 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:56:41 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:56:41 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:56:44 INFO  State of current operation: FAILED
11:56:44 INFO  Assert that (False == value) with value=False ... passed
11:56:44 INFO  Executed sudo, exit status 0
11:56:45 INFO  Executed sudo, exit status 0
11:56:46 INFO  Executed sudo, exit status 0
11:56:50 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:56:50 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:56:50 INFO  Delaying execution by 8 seconds
11:57:12 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:57:12 INFO  Response status: 201
11:57:12 INFO  Response to action: {
    "creationTime": "2022-04-28T11:57:12.253Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901212",
    "id": "17901212",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
11:57:12 INFO  Started operation: {'creationTime': '2022-04-28T11:57:12.253Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901212', 'id': '17901212', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
11:57:12 WARN  , retrying in 0.5 seconds...
11:57:12 WARN  , retrying in 0.5 seconds...
11:57:13 WARN  , retrying in 0.5 seconds...
11:57:13 WARN  , retrying in 0.5 seconds...
11:57:14 WARN  , retrying in 0.5 seconds...
11:57:14 WARN  , retrying in 0.5 seconds...
11:57:15 WARN  , retrying in 0.5 seconds...
11:57:15 WARN  , retrying in 0.5 seconds...
11:57:16 WARN  , retrying in 0.5 seconds...
11:57:16 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
11:57:16 WARN  AssertionError: ... blocked 
11:57:16 INFO  
11:57:16 INFO  cleanup:
11:57:17 INFO  Executed sudo, exit status 0
11:57:17 INFO  Executed sudo, exit status 0
11:57:22 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:57:22 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
11:57:22 INFO  Delaying execution by 8 seconds
11:57:44 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:57:48 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:57:49 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:57:49 INFO  
11:57:49 INFO  Test duration: 84.75 secs
11:57:49 INFO  Test final outcome:  BLOCKED
11:57:49 INFO  Test outcome reason: AssertionError:
11:57:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_fail
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for a failing installation
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_remove
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>11:43:01 INFO  ==============================================================
11:43:01 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
11:43:01 INFO  Dir:   software_management_end_to_end/
11:43:01 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
11:43:01 INFO  ==============================================================
11:43:01 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:43:01 INFO  EnvironmentC8y Setup
11:43:01 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:43:01 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:43:01 INFO  Delaying execution by 6 seconds
11:43:21 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:43:22 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:43:22 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:43:23 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:43:25 INFO  State of current operation: SUCCESSFUL
11:43:25 INFO  Assert that (False == value) with value=False ... passed
11:43:26 INFO  Assert that (False == value) with value=False ... passed
11:43:26 INFO  Assert that (False == value) with value=False ... passed
11:43:26 INFO  Package robotfindskitten is installed
11:43:26 WARN  Assert that (False == value) with value=True ... failed 
11:43:26 INFO  Response status: 201
11:43:26 INFO  Response to action: {
    "creationTime": "2022-04-28T11:43:26.769Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901148",
    "id": "17901148",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:43:26 INFO  Started operation: {'creationTime': '2022-04-28T11:43:26.769Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901148', 'id': '17901148', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:43:29 INFO  State of operation 17901148 : PENDING
11:43:31 INFO  State of operation 17901148 : EXECUTING
11:43:33 INFO  State of operation 17901148 : EXECUTING
11:43:35 INFO  State of operation 17901148 : EXECUTING
11:43:38 INFO  State of operation 17901148 : EXECUTING
11:43:40 INFO  State of operation 17901148 : EXECUTING
11:43:43 INFO  State of operation 17901148 : EXECUTING
11:43:45 INFO  State of operation 17901148 : EXECUTING
11:43:48 INFO  State of operation 17901148 : EXECUTING
11:43:50 INFO  State of operation 17901148 : EXECUTING
11:43:52 INFO  State of operation 17901148 : EXECUTING
11:43:55 INFO  State of operation 17901148 : EXECUTING
11:43:57 INFO  State of operation 17901148 : EXECUTING
11:43:59 INFO  State of operation 17901148 : EXECUTING
11:44:01 INFO  State of operation 17901148 : EXECUTING
11:44:04 INFO  State of operation 17901148 : EXECUTING
11:44:06 INFO  State of operation 17901148 : EXECUTING
11:44:09 INFO  State of operation 17901148 : EXECUTING
11:44:11 INFO  State of operation 17901148 : EXECUTING
11:44:13 INFO  State of operation 17901148 : EXECUTING
11:44:15 INFO  State of operation 17901148 : EXECUTING
11:44:18 INFO  State of operation 17901148 : EXECUTING
11:44:20 INFO  State of operation 17901148 : EXECUTING
11:44:23 INFO  State of operation 17901148 : EXECUTING
11:44:25 INFO  State of operation 17901148 : EXECUTING
11:44:28 INFO  State of operation 17901148 : EXECUTING
11:44:31 INFO  State of operation 17901148 : EXECUTING
11:44:33 INFO  State of operation 17901148 : EXECUTING
11:44:35 INFO  State of operation 17901148 : EXECUTING
11:44:38 INFO  State of operation 17901148 : EXECUTING
11:44:40 INFO  State of operation 17901148 : EXECUTING
11:44:42 INFO  State of operation 17901148 : EXECUTING
11:44:45 INFO  State of operation 17901148 : EXECUTING
11:44:47 INFO  State of operation 17901148 : SUCCESSFUL
11:44:47 INFO  Assert that (False == value) with value=False ... passed
11:44:47 INFO  Package asciijump is installed
11:44:47 INFO  Assert that (True == value) with value=True ... passed
11:44:48 INFO  Package squirrel3 is installed
11:44:48 INFO  Assert that (True == value) with value=True ... passed
11:44:48 INFO  Package robotfindskitten is installed
11:44:48 INFO  Assert that (True == value) with value=True ... passed
11:44:48 INFO  Response status: 201
11:44:48 INFO  Response to action: {
    "creationTime": "2022-04-28T11:44:48.424Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901156",
    "id": "17901156",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:44:48 INFO  Started operation: {'creationTime': '2022-04-28T11:44:48.424Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901156', 'id': '17901156', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
11:44:50 INFO  State of operation 17901156 : PENDING
11:44:52 INFO  State of operation 17901156 : EXECUTING
11:44:55 INFO  State of operation 17901156 : EXECUTING
11:44:57 INFO  State of operation 17901156 : EXECUTING
11:44:59 INFO  State of operation 17901156 : EXECUTING
11:45:01 INFO  State of operation 17901156 : EXECUTING
11:45:04 INFO  State of operation 17901156 : EXECUTING
11:45:07 INFO  State of operation 17901156 : EXECUTING
11:45:09 INFO  State of operation 17901156 : EXECUTING
11:45:11 INFO  State of operation 17901156 : EXECUTING
11:45:14 INFO  State of operation 17901156 : EXECUTING
11:45:16 INFO  State of operation 17901156 : EXECUTING
11:45:18 INFO  State of operation 17901156 : EXECUTING
11:45:20 INFO  State of operation 17901156 : EXECUTING
11:45:23 INFO  State of operation 17901156 : EXECUTING
11:45:25 INFO  State of operation 17901156 : EXECUTING
11:45:28 INFO  State of operation 17901156 : EXECUTING
11:45:30 INFO  State of operation 17901156 : EXECUTING
11:45:33 INFO  State of operation 17901156 : EXECUTING
11:45:35 INFO  State of operation 17901156 : EXECUTING
11:45:37 INFO  State of operation 17901156 : EXECUTING
11:45:39 INFO  State of operation 17901156 : EXECUTING
11:45:41 INFO  State of operation 17901156 : EXECUTING
11:45:44 INFO  State of operation 17901156 : EXECUTING
11:45:46 INFO  State of operation 17901156 : EXECUTING
11:45:49 INFO  State of operation 17901156 : EXECUTING
11:45:51 INFO  State of operation 17901156 : EXECUTING
11:45:53 INFO  State of operation 17901156 : SUCCESSFUL
11:45:54 INFO  Assert that (False == value) with value=False ... passed
11:45:54 INFO  Assert that (False == value) with value=False ... passed
11:45:54 INFO  Assert that (False == value) with value=False ... passed
11:45:54 INFO  Assert that (False == value) with value=False ... passed
11:45:54 INFO  
11:45:54 INFO  cleanup:
11:45:58 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:45:59 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:45:59 INFO  
11:45:59 INFO  Test duration: 178.13 secs
11:45:59 INFO  Test final outcome:  FAILED
11:45:59 INFO  Test outcome reason: Assert that (False == value) with value=True
11:45:59 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>12:04:09 INFO  ==============================================================
12:04:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
12:04:09 INFO  Dir:   software_management_end_to_end/
12:04:09 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
12:04:09 INFO  ==============================================================
12:04:09 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:04:09 INFO  EnvironmentC8y Setup
12:04:09 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:04:09 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
12:04:09 INFO  Delaying execution by 6 seconds
12:04:30 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:04:30 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:04:31 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:04:31 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:04:33 INFO  State of current operation: SUCCESSFUL
12:04:34 INFO  Assert that (False == value) with value=False ... passed
12:04:34 INFO  Package asciijump is installed
12:04:34 WARN  Assert that (False == value) with value=True ... failed 
12:04:34 INFO  Package squirrel3 is installed
12:04:34 WARN  Assert that (False == value) with value=True ... failed 
12:04:35 INFO  Package robotfindskitten is installed
12:04:35 WARN  Assert that (False == value) with value=True ... failed 
12:04:35 INFO  Response status: 201
12:04:35 INFO  Response to action: {
    "creationTime": "2022-04-28T12:04:35.239Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901295",
    "id": "17901295",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
12:04:35 INFO  Started operation: {'creationTime': '2022-04-28T12:04:35.239Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901295', 'id': '17901295', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
12:04:37 INFO  State of operation 17901295 : PENDING
12:04:39 INFO  State of operation 17901295 : EXECUTING
12:04:41 INFO  State of operation 17901295 : EXECUTING
12:04:44 INFO  State of operation 17901295 : EXECUTING
12:04:46 INFO  State of operation 17901295 : EXECUTING
12:04:48 INFO  State of operation 17901295 : EXECUTING
12:04:51 INFO  State of operation 17901295 : EXECUTING
12:04:54 INFO  State of operation 17901295 : EXECUTING
12:04:56 INFO  State of operation 17901295 : EXECUTING
12:04:58 INFO  State of operation 17901295 : EXECUTING
12:05:01 INFO  State of operation 17901295 : EXECUTING
12:05:03 INFO  State of operation 17901295 : EXECUTING
12:05:05 INFO  State of operation 17901295 : EXECUTING
12:05:07 INFO  State of operation 17901295 : EXECUTING
12:05:09 INFO  State of operation 17901295 : EXECUTING
12:05:12 INFO  State of operation 17901295 : SUCCESSFUL
12:05:12 INFO  Assert that (False == value) with value=False ... passed
12:05:12 INFO  Package asciijump is installed
12:05:12 INFO  Assert that (True == value) with value=True ... passed
12:05:13 INFO  Package squirrel3 is installed
12:05:13 INFO  Assert that (True == value) with value=True ... passed
12:05:13 INFO  Package robotfindskitten is installed
12:05:13 INFO  Assert that (True == value) with value=True ... passed
12:05:13 INFO  Response status: 201
12:05:13 INFO  Response to action: {
    "creationTime": "2022-04-28T12:05:13.468Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901307",
    "id": "17901307",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
12:05:13 INFO  Started operation: {'creationTime': '2022-04-28T12:05:13.468Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901307', 'id': '17901307', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
12:05:15 INFO  State of operation 17901307 : PENDING
12:05:18 INFO  State of operation 17901307 : EXECUTING
12:05:20 INFO  State of operation 17901307 : EXECUTING
12:05:22 INFO  State of operation 17901307 : EXECUTING
12:05:24 INFO  State of operation 17901307 : EXECUTING
12:05:26 INFO  State of operation 17901307 : EXECUTING
12:05:29 INFO  State of operation 17901307 : EXECUTING
12:05:32 INFO  State of operation 17901307 : EXECUTING
12:05:34 INFO  State of operation 17901307 : EXECUTING
12:05:37 INFO  State of operation 17901307 : EXECUTING
12:05:39 INFO  State of operation 17901307 : EXECUTING
12:05:41 INFO  State of operation 17901307 : EXECUTING
12:05:43 INFO  State of operation 17901307 : EXECUTING
12:05:46 INFO  State of operation 17901307 : EXECUTING
12:05:48 INFO  State of operation 17901307 : EXECUTING
12:05:50 INFO  State of operation 17901307 : EXECUTING
12:05:53 INFO  State of operation 17901307 : EXECUTING
12:05:56 INFO  State of operation 17901307 : EXECUTING
12:05:58 INFO  State of operation 17901307 : EXECUTING
12:06:00 INFO  State of operation 17901307 : EXECUTING
12:06:02 INFO  State of operation 17901307 : EXECUTING
12:06:05 INFO  State of operation 17901307 : EXECUTING
12:06:07 INFO  State of operation 17901307 : EXECUTING
12:06:09 INFO  State of operation 17901307 : EXECUTING
12:06:12 INFO  State of operation 17901307 : EXECUTING
12:06:14 INFO  State of operation 17901307 : EXECUTING
12:06:16 INFO  State of operation 17901307 : EXECUTING
12:06:19 INFO  State of operation 17901307 : EXECUTING
12:06:21 INFO  State of operation 17901307 : SUCCESSFUL
12:06:21 INFO  Assert that (False == value) with value=False ... passed
12:06:21 INFO  Assert that (False == value) with value=False ... passed
12:06:22 INFO  Assert that (False == value) with value=False ... passed
12:06:22 INFO  Assert that (False == value) with value=False ... passed
12:06:22 INFO  
12:06:22 INFO  cleanup:
12:06:26 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:06:27 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:06:27 INFO  
12:06:27 INFO  Test duration: 137.93 secs
12:06:27 INFO  Test final outcome:  FAILED
12:06:27 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
12:06:27 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>11:45:59 INFO  ==============================================================
11:45:59 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
11:45:59 INFO  Dir:   software_management_end_to_end/
11:45:59 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
11:45:59 INFO  ==============================================================
11:45:59 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:45:59 INFO  EnvironmentC8y Setup
11:46:00 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:46:00 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
11:46:00 INFO  Delaying execution by 6 seconds
11:46:20 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
11:46:20 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:46:21 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:46:21 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:46:23 INFO  State of current operation: SUCCESSFUL
11:46:24 INFO  Assert that (False == value) with value=False ... passed
11:46:24 INFO  Assert that (False == value) with value=False ... passed
11:46:24 INFO  Assert that (False == value) with value=False ... passed
11:46:24 INFO  Assert that (False == value) with value=False ... passed
11:46:24 INFO  Assert that (False == value) with value=False ... passed
11:46:26 INFO  Response status: 201
11:46:26 INFO  Response to action: {
    "creationTime": "2022-04-28T11:46:26.108Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901167",
    "id": "17901167",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
11:46:26 INFO  Started operation: {'creationTime': '2022-04-28T11:46:26.108Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901167', 'id': '17901167', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
11:46:28 INFO  State of operation 17901167 : PENDING
11:46:30 INFO  State of operation 17901167 : EXECUTING
11:46:32 INFO  State of operation 17901167 : EXECUTING
11:46:35 INFO  State of operation 17901167 : EXECUTING
11:46:37 INFO  State of operation 17901167 : EXECUTING
11:46:39 INFO  State of operation 17901167 : EXECUTING
11:46:42 INFO  State of operation 17901167 : EXECUTING
11:46:44 INFO  State of operation 17901167 : EXECUTING
11:46:47 INFO  State of operation 17901167 : EXECUTING
11:46:49 INFO  State of operation 17901167 : EXECUTING
11:46:51 INFO  State of operation 17901167 : EXECUTING
11:46:54 INFO  State of operation 17901167 : EXECUTING
11:46:56 INFO  State of operation 17901167 : EXECUTING
11:46:58 INFO  State of operation 17901167 : EXECUTING
11:47:00 INFO  State of operation 17901167 : EXECUTING
11:47:03 INFO  State of operation 17901167 : EXECUTING
11:47:05 INFO  State of operation 17901167 : EXECUTING
11:47:08 INFO  State of operation 17901167 : EXECUTING
11:47:10 INFO  State of operation 17901167 : EXECUTING
11:47:12 INFO  State of operation 17901167 : EXECUTING
11:47:14 INFO  State of operation 17901167 : EXECUTING
11:47:17 INFO  State of operation 17901167 : EXECUTING
11:47:20 INFO  State of operation 17901167 : EXECUTING
11:47:22 INFO  State of operation 17901167 : EXECUTING
11:47:24 INFO  State of operation 17901167 : EXECUTING
11:47:26 INFO  State of operation 17901167 : EXECUTING
11:47:29 INFO  State of operation 17901167 : EXECUTING
11:47:31 INFO  State of operation 17901167 : EXECUTING
11:47:34 INFO  State of operation 17901167 : EXECUTING
11:47:36 INFO  State of operation 17901167 : EXECUTING
11:47:38 INFO  State of operation 17901167 : EXECUTING
11:47:40 INFO  State of operation 17901167 : EXECUTING
11:47:43 INFO  State of operation 17901167 : EXECUTING
11:47:45 INFO  State of operation 17901167 : EXECUTING
11:47:47 INFO  State of operation 17901167 : EXECUTING
11:47:49 INFO  State of operation 17901167 : EXECUTING
11:47:52 INFO  State of operation 17901167 : EXECUTING
11:47:54 INFO  State of operation 17901167 : EXECUTING
11:47:57 INFO  State of operation 17901167 : EXECUTING
11:47:59 INFO  State of operation 17901167 : EXECUTING
11:48:02 INFO  State of operation 17901167 : EXECUTING
11:48:04 INFO  State of operation 17901167 : EXECUTING
11:48:06 INFO  State of operation 17901167 : EXECUTING
11:48:08 INFO  State of operation 17901167 : EXECUTING
11:48:11 INFO  State of operation 17901167 : EXECUTING
11:48:13 INFO  State of operation 17901167 : EXECUTING
11:48:15 INFO  State of operation 17901167 : EXECUTING
11:48:17 INFO  State of operation 17901167 : EXECUTING
11:48:20 INFO  State of operation 17901167 : EXECUTING
11:48:22 INFO  State of operation 17901167 : EXECUTING
11:48:25 INFO  State of operation 17901167 : EXECUTING
11:48:27 INFO  State of operation 17901167 : EXECUTING
11:48:29 INFO  State of operation 17901167 : EXECUTING
11:48:32 INFO  State of operation 17901167 : EXECUTING
11:48:34 INFO  State of operation 17901167 : EXECUTING
11:48:37 INFO  State of operation 17901167 : EXECUTING
11:48:39 INFO  State of operation 17901167 : EXECUTING
11:48:41 INFO  State of operation 17901167 : EXECUTING
11:48:43 INFO  State of operation 17901167 : EXECUTING
11:48:45 INFO  State of operation 17901167 : EXECUTING
11:48:48 INFO  State of operation 17901167 : EXECUTING
11:48:50 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
11:48:50 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
11:48:50 INFO  
11:48:50 INFO  cleanup:
11:48:56 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:48:57 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:48:57 INFO  
11:48:57 INFO  Test duration: 177.95 secs
11:48:57 INFO  Test final outcome:  BLOCKED
11:48:57 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
11:48:57 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>12:06:27 INFO  ==============================================================
12:06:27 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
12:06:27 INFO  Dir:   software_management_end_to_end/
12:06:27 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
12:06:27 INFO  ==============================================================
12:06:27 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:06:27 INFO  EnvironmentC8y Setup
12:06:27 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:06:27 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
12:06:27 INFO  Delaying execution by 6 seconds
12:06:47 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
12:06:48 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:06:48 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:06:49 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:06:51 INFO  State of current operation: SUCCESSFUL
12:06:51 INFO  Assert that (False == value) with value=False ... passed
12:06:52 INFO  Assert that (False == value) with value=False ... passed
12:06:52 INFO  Assert that (False == value) with value=False ... passed
12:06:52 INFO  Assert that (False == value) with value=False ... passed
12:06:52 INFO  Package moon-buggy is installed
12:06:52 WARN  Assert that (False == value) with value=True ... failed 
12:06:54 INFO  Response status: 201
12:06:54 INFO  Response to action: {
    "creationTime": "2022-04-28T12:06:54.021Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901321",
    "id": "17901321",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
12:06:54 INFO  Started operation: {'creationTime': '2022-04-28T12:06:54.021Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901321', 'id': '17901321', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
12:06:56 INFO  State of operation 17901321 : PENDING
12:06:58 INFO  State of operation 17901321 : EXECUTING
12:07:00 INFO  State of operation 17901321 : EXECUTING
12:07:02 INFO  State of operation 17901321 : EXECUTING
12:07:05 INFO  State of operation 17901321 : EXECUTING
12:07:07 INFO  State of operation 17901321 : EXECUTING
12:07:10 INFO  State of operation 17901321 : EXECUTING
12:07:12 INFO  State of operation 17901321 : EXECUTING
12:07:15 INFO  State of operation 17901321 : EXECUTING
12:07:17 INFO  State of operation 17901321 : EXECUTING
12:07:19 INFO  State of operation 17901321 : EXECUTING
12:07:21 INFO  State of operation 17901321 : EXECUTING
12:07:24 INFO  State of operation 17901321 : EXECUTING
12:07:26 INFO  State of operation 17901321 : EXECUTING
12:07:28 INFO  State of operation 17901321 : EXECUTING
12:07:31 INFO  State of operation 17901321 : EXECUTING
12:07:33 INFO  State of operation 17901321 : EXECUTING
12:07:35 INFO  State of operation 17901321 : EXECUTING
12:07:38 INFO  State of operation 17901321 : EXECUTING
12:07:40 INFO  State of operation 17901321 : EXECUTING
12:07:42 INFO  State of operation 17901321 : EXECUTING
12:07:45 INFO  State of operation 17901321 : EXECUTING
12:07:47 INFO  State of operation 17901321 : EXECUTING
12:07:49 INFO  State of operation 17901321 : EXECUTING
12:07:52 INFO  State of operation 17901321 : EXECUTING
12:07:54 INFO  State of operation 17901321 : EXECUTING
12:07:56 INFO  State of operation 17901321 : EXECUTING
12:07:59 INFO  State of operation 17901321 : EXECUTING
12:08:01 INFO  State of operation 17901321 : EXECUTING
12:08:03 INFO  State of operation 17901321 : EXECUTING
12:08:06 INFO  State of operation 17901321 : EXECUTING
12:08:08 INFO  State of operation 17901321 : EXECUTING
12:08:10 INFO  State of operation 17901321 : EXECUTING
12:08:13 INFO  State of operation 17901321 : EXECUTING
12:08:15 INFO  State of operation 17901321 : EXECUTING
12:08:17 INFO  State of operation 17901321 : EXECUTING
12:08:19 INFO  State of operation 17901321 : EXECUTING
12:08:21 INFO  State of operation 17901321 : EXECUTING
12:08:24 INFO  State of operation 17901321 : EXECUTING
12:08:27 INFO  State of operation 17901321 : EXECUTING
12:08:29 INFO  State of operation 17901321 : EXECUTING
12:08:32 INFO  State of operation 17901321 : EXECUTING
12:08:34 INFO  State of operation 17901321 : EXECUTING
12:08:36 INFO  State of operation 17901321 : EXECUTING
12:08:38 INFO  State of operation 17901321 : EXECUTING
12:08:40 INFO  State of operation 17901321 : EXECUTING
12:08:43 INFO  State of operation 17901321 : EXECUTING
12:08:45 INFO  State of operation 17901321 : EXECUTING
12:08:48 INFO  State of operation 17901321 : EXECUTING
12:08:50 INFO  State of operation 17901321 : EXECUTING
12:08:52 INFO  State of operation 17901321 : EXECUTING
12:08:54 INFO  State of operation 17901321 : EXECUTING
12:08:57 INFO  State of operation 17901321 : EXECUTING
12:08:59 INFO  State of operation 17901321 : EXECUTING
12:09:02 INFO  State of operation 17901321 : EXECUTING
12:09:04 INFO  State of operation 17901321 : EXECUTING
12:09:07 INFO  State of operation 17901321 : EXECUTING
12:09:09 INFO  State of operation 17901321 : EXECUTING
12:09:11 INFO  State of operation 17901321 : EXECUTING
12:09:13 INFO  State of operation 17901321 : EXECUTING
12:09:16 INFO  State of operation 17901321 : EXECUTING
12:09:18 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
12:09:18 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
12:09:18 INFO  
12:09:18 INFO  cleanup:
12:09:22 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:09:23 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:09:23 INFO  
12:09:23 INFO  Test duration: 176.18 secs
12:09:23 INFO  Test final outcome:  BLOCKED
12:09:23 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
12:09:23 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.03 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>11:53:40 INFO  ==============================================================
11:53:40 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
11:53:40 INFO  Dir:   software_management_end_to_end/
11:53:40 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
11:53:40 INFO  ==============================================================
11:53:41 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
11:53:41 INFO  EnvironmentC8y Setup
11:53:41 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
11:53:56 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:53:56 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:53:57 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:53:57 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:53:59 INFO  State of current operation: SUCCESSFUL
11:54:00 INFO  Response status: 201
11:54:00 INFO  Response to action: {
    "creationTime": "2022-04-28T11:54:00.012Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901199",
    "id": "17901199",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
11:54:00 INFO  Started operation: {'creationTime': '2022-04-28T11:54:00.012Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901199', 'id': '17901199', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
11:54:02 INFO  State of operation 17901199 : PENDING
11:54:04 INFO  State of operation 17901199 : EXECUTING
11:54:06 INFO  State of operation 17901199 : EXECUTING
11:54:08 INFO  State of operation 17901199 : EXECUTING
11:54:11 INFO  State of operation 17901199 : FAILED
11:54:11 ERROR Stopping as the operation has failed
11:54:11 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
11:54:11 WARN  SystemError: The operation has failed ... blocked 
11:54:11 INFO  
11:54:11 INFO  cleanup:
11:54:15 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:54:15 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:54:15 INFO  
11:54:15 INFO  Test duration: 35.62 secs
11:54:15 INFO  Test final outcome:  BLOCKED
11:54:15 INFO  Test outcome reason: SystemError: The operation has failed
11:54:15 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>12:12:00 INFO  ==============================================================
12:12:00 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
12:12:00 INFO  Dir:   software_management_end_to_end/
12:12:00 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
12:12:00 INFO  ==============================================================
12:12:00 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:12:00 INFO  EnvironmentC8y Setup
12:12:01 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:12:01 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
12:12:01 INFO  Delaying execution by 7 seconds
12:12:22 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:12:22 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:12:23 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:12:23 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:12:26 INFO  State of current operation: SUCCESSFUL
12:12:26 INFO  Response status: 201
12:12:26 INFO  Response to action: {
    "creationTime": "2022-04-28T12:12:26.324Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901379",
    "id": "17901379",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
12:12:26 INFO  Started operation: {'creationTime': '2022-04-28T12:12:26.324Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901379', 'id': '17901379', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
12:12:28 INFO  State of operation 17901379 : PENDING
12:12:30 INFO  State of operation 17901379 : EXECUTING
12:12:33 INFO  State of operation 17901379 : EXECUTING
12:12:35 INFO  State of operation 17901379 : EXECUTING
12:12:37 INFO  State of operation 17901379 : FAILED
12:12:37 ERROR Stopping as the operation has failed
12:12:37 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
12:12:37 WARN  SystemError: The operation has failed ... blocked 
12:12:37 INFO  
12:12:37 INFO  cleanup:
12:12:41 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:12:42 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:12:42 INFO  
12:12:42 INFO  Test duration: 41.48 secs
12:12:42 INFO  Test final outcome:  BLOCKED
12:12:42 INFO  Test outcome reason: SystemError: The operation has failed
12:12:42 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_fake_install_remove
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
11:11:47 WARN  Aborted test due to SKIPPED outcome
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
11:11:47 INFO  ==============================================================
11:11:47 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
11:11:47 INFO  
11:11:47 INFO  Test duration: 0.02 secs
11:11:47 INFO  Test final outcome:  SKIPPED
11:11:47 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
11:11:47 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>12:14:15 INFO  ==============================================================
12:14:15 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
12:14:15 INFO  Dir:   software_management_end_to_end/
12:14:15 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
12:14:15 INFO  ==============================================================
12:14:15 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
12:14:15 INFO  
12:14:15 INFO  Test duration: 0.02 secs
12:14:15 INFO  Test final outcome:  SKIPPED
12:14:15 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
12:14:15 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>11:11:47 INFO  ==============================================================
11:11:47 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
11:11:47 INFO  Dir:   software_management_end_to_end/
11:11:47 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
11:11:47 INFO  ==============================================================
11:11:47 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
11:11:48 WARN  Aborted test due to SKIPPED outcome
11:11:48 INFO  
11:11:48 INFO  Test duration: 0.03 secs
11:11:48 INFO  Test final outcome:  SKIPPED
11:11:48 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
11:11:48 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>12:14:15 INFO  ==============================================================
12:14:15 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
12:14:15 INFO  Dir:   software_management_end_to_end/
12:14:15 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
12:14:15 INFO  ==============================================================
12:14:15 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:14:15 INFO  EnvironmentC8y Setup
12:14:16 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:14:16 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
12:14:16 INFO  Delaying execution by 7 seconds
12:14:37 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:14:38 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:14:38 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:14:39 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:14:41 INFO  State of current operation: SUCCESSFUL
12:14:41 INFO  Package apple is installed
12:14:41 INFO  Assert that (True == value) with value=True ... passed
12:14:41 INFO  Package banana is installed
12:14:41 INFO  Assert that (True == value) with value=True ... passed
12:14:42 INFO  Package cherry is installed
12:14:42 INFO  Assert that (True == value) with value=True ... passed
12:14:42 INFO  Package asciijump is installed
12:14:42 WARN  Assert that (False == value) with value=True ... failed 
12:14:42 INFO  Assert that (False == value) with value=False ... passed
12:14:42 INFO  Response status: 201
12:14:42 INFO  Response to action: {
    "creationTime": "2022-04-28T12:14:42.754Z",
    "deviceName": "m32sd10b",
    "deviceId": "17884833",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901438",
    "id": "17901438",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
12:14:42 INFO  Started operation: {'creationTime': '2022-04-28T12:14:42.754Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17901438', 'id': '17901438', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
12:14:45 INFO  State of operation 17901438 : PENDING
12:14:47 INFO  State of operation 17901438 : EXECUTING
12:14:49 INFO  State of operation 17901438 : EXECUTING
12:14:51 INFO  State of operation 17901438 : EXECUTING
12:14:53 INFO  State of operation 17901438 : EXECUTING
12:14:56 INFO  State of operation 17901438 : EXECUTING
12:14:58 INFO  State of operation 17901438 : EXECUTING
12:15:01 INFO  State of operation 17901438 : EXECUTING
12:15:03 INFO  State of operation 17901438 : EXECUTING
12:15:06 INFO  State of operation 17901438 : EXECUTING
12:15:08 INFO  State of operation 17901438 : EXECUTING
12:15:10 INFO  State of operation 17901438 : EXECUTING
12:15:12 INFO  State of operation 17901438 : EXECUTING
12:15:15 INFO  State of operation 17901438 : EXECUTING
12:15:17 INFO  State of operation 17901438 : EXECUTING
12:15:19 INFO  State of operation 17901438 : EXECUTING
12:15:21 INFO  State of operation 17901438 : FAILED
12:15:21 ERROR Stopping as the operation has failed
12:15:21 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
12:15:21 WARN  SystemError: The operation has failed ... blocked 
12:15:21 INFO  
12:15:21 INFO  cleanup:
12:15:25 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:15:26 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:15:26 INFO  
12:15:26 INFO  Test duration: 70.67 secs
12:15:26 INFO  Test final outcome:  BLOCKED
12:15:26 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
12:15:26 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session" />
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_fail_noarg
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.05 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_finalize
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin finalize - dense version
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.02 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_install
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin prepare - dense version
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.02 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_install_fail
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin prepare - failing
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.02 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_install_local_file
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin install from local file - SUCCESS case
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.02 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_install_local_file_fail
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin install from local file - FAIL case
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.03 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_install_local_file_with_version
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.03 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:11 INFO  ==============================================================
11:02:11 INFO  Id:    apt_install_local_file_with_version_fail
11:02:11 INFO  Dir:   plugin_apt/
11:02:11 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
11:02:11 INFO  ==============================================================
11:02:11 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:11 WARN  Aborted test due to SKIPPED outcome
11:02:11 INFO  
11:02:11 INFO  Test duration: 0.03 secs
11:02:11 INFO  Test final outcome:  SKIPPED
11:02:11 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:11 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_prepare
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin install - dense version
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.04 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_prepare_install_finalize
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin install use case
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_remove
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin prepare - dense version
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_remove_fail
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin prepare - failing
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_sequence_install_with_version
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin install with version use case
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_sequence_install_with_version_fail
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin install with version fails use case
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_sequence_remove
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin prepare install remove finalize use case
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_sequence_remove_with_version
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin remove with version use case
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_sequence_remove_with_version_fail
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin remove with version fails when the version not found
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_update_list
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin update-list command use case
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    apt_wrong_option
11:02:15 INFO  Dir:   plugin_apt/
11:02:15 INFO  Title: Validate apt plugin install
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval">
		<failure message="FAILED: Grep on resp_jwt.out contains &quot;\[c8y/s/dat\] 71,&quot;" type="FAILED" />
		<system-out>10:58:29 INFO  ==============================================================
10:58:29 INFO  Id:    c8y_jwt_token_retrieval
10:58:29 INFO  Dir:   cloud_c8y/
10:58:29 INFO  Title: Validate the Cumulocity JWT Token Retrieval feature
10:58:29 INFO  ==============================================================
10:58:30 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:58:45 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:58:45 INFO  Started sudo&lt;resp_jwt&gt; with process id 14681
10:58:46 INFO  Executed sudo&lt;req_jwt&gt;, exit status 0
10:58:48 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
10:58:48 WARN  Grep on resp_jwt.out contains "\[c8y/s/dat\] 71," ... failed [run.py:60]
10:58:48 INFO  
10:58:48 INFO  cleanup:
10:58:53 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:58:53 INFO  
10:58:53 INFO  Test duration: 23.55 secs
10:58:53 INFO  Test final outcome:  FAILED
10:58:53 INFO  Test outcome reason: Grep on resp_jwt.out contains "\[c8y/s/dat\] 71,"
10:58:53 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    docker_plugin_finalize
11:02:15 INFO  Dir:   plugin_docker/
11:02:15 INFO  Title: Validate docker plugin finalize command
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.05 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<failure message="BLOCKED: sudo&lt;docker_pull&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>11:53:08 INFO  ==============================================================
11:53:08 INFO  Id:    docker_plugin_finalize
11:53:08 INFO  Dir:   plugin_docker/
11:53:08 INFO  Title: Validate docker plugin finalize command
11:53:08 INFO  ==============================================================
11:53:10 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:53:10 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
11:53:12 WARN  Executed sudo&lt;docker_pull&gt;, exit status 1
11:53:12 INFO  Contents of docker_pull.err: 
  Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  -----

11:53:13 WARN  sudo&lt;docker_pull&gt; returned non-zero exit code 1 ... blocked 
11:53:13 WARN  Aborted test due to BLOCKED outcome
11:53:13 INFO  
11:53:13 INFO  cleanup:
11:53:13 INFO  
11:53:13 INFO  Test duration: 4.78 secs
11:53:13 INFO  Test final outcome:  BLOCKED
11:53:13 INFO  Test outcome reason: sudo&lt;docker_pull&gt; returned non-zero exit code 1
11:53:13 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    docker_plugin_install
11:02:15 INFO  Dir:   plugin_docker/
11:02:15 INFO  Title: Validate docker plugin install
11:02:15 INFO  ==============================================================
11:02:15 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:02:15 WARN  Aborted test due to SKIPPED outcome
11:02:15 INFO  
11:02:15 INFO  Test duration: 0.02 secs
11:02:15 INFO  Test final outcome:  SKIPPED
11:02:15 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:02:15 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>11:53:13 INFO  ==============================================================
11:53:13 INFO  Id:    docker_plugin_install
11:53:13 INFO  Dir:   plugin_docker/
11:53:13 INFO  Title: Validate docker plugin install
11:53:13 INFO  ==============================================================
11:53:13 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:53:13 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_images.out ... passed
11:53:16 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:53:16 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:23]
11:53:17 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
11:53:18 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
11:53:18 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_ps.out contains "registry" ... failed [environment_docker_plugin.py:49,run.py:26]
11:53:18 INFO  
11:53:18 INFO  cleanup:
11:53:18 INFO  Removing containers scheduled to be cleaned up: ['']
11:53:18 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
11:53:18 INFO  Removing images scheduled to be cleaned up: ['registry']
11:53:19 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
11:53:19 INFO  
11:53:19 INFO  Test duration: 6.55 secs
11:53:19 INFO  Test final outcome:  BLOCKED
11:53:19 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
11:53:19 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:02:15 INFO  ==============================================================
11:02:15 INFO  Id:    docker_plugin_install_with_version
11:02:15 INFO  Dir:   plugin_docker/
11:02:15 INFO  Title: Validate docker plugin install
11:02:15 INFO  ==============================================================
11:02:16 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:02:16 WARN  Aborted test due to SKIPPED outcome
11:02:16 INFO  
11:02:16 INFO  Test duration: 0.07 secs
11:02:16 INFO  Test final outcome:  SKIPPED
11:02:16 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:02:16 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>11:53:19 INFO  ==============================================================
11:53:19 INFO  Id:    docker_plugin_install_with_version
11:53:19 INFO  Dir:   plugin_docker/
11:53:19 INFO  Title: Validate docker plugin install
11:53:19 INFO  ==============================================================
11:53:20 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:53:20 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
11:53:22 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
11:53:22 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:24]
11:53:23 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
11:53:24 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
11:53:24 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:28]
11:53:24 INFO  
11:53:24 INFO  cleanup:
11:53:24 INFO  Removing containers scheduled to be cleaned up: ['', '']
11:53:25 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
11:53:25 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
11:53:26 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
11:53:26 INFO  
11:53:26 INFO  Test duration: 6.42 secs
11:53:26 INFO  Test final outcome:  BLOCKED
11:53:26 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
11:53:26 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:02:16 INFO  ==============================================================
11:02:16 INFO  Id:    docker_plugin_remove
11:02:16 INFO  Dir:   plugin_docker/
11:02:16 INFO  Title: Validate docker plugin remove
11:02:16 INFO  ==============================================================
11:02:16 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:02:16 WARN  Aborted test due to SKIPPED outcome
11:02:16 INFO  
11:02:16 INFO  Test duration: 0.02 secs
11:02:16 INFO  Test final outcome:  SKIPPED
11:02:16 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:02:16 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>11:53:26 INFO  ==============================================================
11:53:26 INFO  Id:    docker_plugin_remove
11:53:26 INFO  Dir:   plugin_docker/
11:53:26 INFO  Title: Validate docker plugin remove
11:53:26 INFO  ==============================================================
11:53:26 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:53:26 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
11:53:29 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
11:53:29 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

11:53:29 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
11:53:29 WARN  Aborted test due to BLOCKED outcome
11:53:29 INFO  
11:53:29 INFO  cleanup:
11:53:29 INFO  Removing containers scheduled to be cleaned up: ['', '']
11:53:29 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
11:53:29 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
11:53:30 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
11:53:30 INFO  
11:53:30 INFO  Test duration: 4.65 secs
11:53:30 INFO  Test final outcome:  BLOCKED
11:53:30 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
11:53:30 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:02:16 INFO  ==============================================================
11:02:16 INFO  Id:    docker_plugin_remove_with_version
11:02:16 INFO  Dir:   plugin_docker/
11:02:16 INFO  Title: Validate docker plugin remove with version
11:02:16 INFO  ==============================================================
11:02:16 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:02:16 WARN  Aborted test due to SKIPPED outcome
11:02:16 INFO  
11:02:16 INFO  Test duration: 0.02 secs
11:02:16 INFO  Test final outcome:  SKIPPED
11:02:16 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:02:16 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>11:53:30 INFO  ==============================================================
11:53:30 INFO  Id:    docker_plugin_remove_with_version
11:53:30 INFO  Dir:   plugin_docker/
11:53:30 INFO  Title: Validate docker plugin remove with version
11:53:30 INFO  ==============================================================
11:53:31 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:53:31 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
11:53:33 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
11:53:33 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

11:53:33 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
11:53:33 WARN  Aborted test due to BLOCKED outcome
11:53:33 INFO  
11:53:33 INFO  cleanup:
11:53:33 INFO  Removing containers scheduled to be cleaned up: ['', '']
11:53:34 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
11:53:34 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
11:53:35 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
11:53:35 INFO  
11:53:35 INFO  Test duration: 4.59 secs
11:53:35 INFO  Test final outcome:  BLOCKED
11:53:35 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
11:53:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>11:02:16 INFO  ==============================================================
11:02:16 INFO  Id:    docker_plugin_update
11:02:16 INFO  Dir:   plugin_docker/
11:02:16 INFO  Title: Validate docker plugin install updates existing containers
11:02:16 INFO  ==============================================================
11:02:16 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
11:02:16 WARN  Aborted test due to SKIPPED outcome
11:02:16 INFO  
11:02:16 INFO  Test duration: 0.02 secs
11:02:16 INFO  Test final outcome:  SKIPPED
11:02:16 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
11:02:16 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>11:53:35 INFO  ==============================================================
11:53:35 INFO  Id:    docker_plugin_update
11:53:35 INFO  Dir:   plugin_docker/
11:53:35 INFO  Title: Validate docker plugin install updates existing containers
11:53:35 INFO  ==============================================================
11:53:36 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
11:53:36 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
11:53:38 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
11:53:38 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.6.2' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

11:53:38 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
11:53:38 WARN  Aborted test due to BLOCKED outcome
11:53:38 INFO  
11:53:38 INFO  cleanup:
11:53:38 INFO  Removing containers scheduled to be cleaned up: ['', '']
11:53:39 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
11:53:39 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
11:53:40 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
11:53:40 INFO  
11:53:40 INFO  Test duration: 4.59 secs
11:53:40 INFO  Test final outcome:  BLOCKED
11:53:40 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
11:53:40 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo" />
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>10:59:12 INFO  ==============================================================
10:59:12 INFO  Id:    log_request_error_messages
10:59:12 INFO  Dir:   log_request_end_to_end/
10:59:12 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
10:59:12 INFO  ==============================================================
10:59:14 INFO  EnvironmentC8y Setup
10:59:14 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:59:14 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 2 seconds
10:59:14 INFO  Delaying execution by 9 seconds
10:59:37 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:59:38 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:59:38 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:59:39 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:59:40 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
10:59:40 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
10:59:41 INFO  log response None
10:59:41 WARN  retry, retrying in 1 seconds...
10:59:42 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/17885600/binaries
10:59:43 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
10:59:43 INFO  
10:59:43 INFO  cleanup:
10:59:43 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
10:59:44 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
10:59:44 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
10:59:48 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:59:49 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:59:49 INFO  
10:59:49 INFO  Test duration: 36.25 secs
10:59:49 INFO  Test final outcome:  FAILED
10:59:49 INFO  Test outcome reason: Assert that (True == value) with value=False
10:59:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>10:59:49 INFO  ==============================================================
10:59:49 INFO  Id:    log_request_num_lines
10:59:49 INFO  Dir:   log_request_end_to_end/
10:59:49 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
10:59:49 INFO  ==============================================================
10:59:49 INFO  EnvironmentC8y Setup
10:59:49 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:59:50 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:59:50 INFO  Delaying execution by 6 seconds
11:00:10 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
11:00:11 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
11:00:11 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
11:00:12 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
11:00:12 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
11:00:13 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
11:00:13 INFO  log response None
11:00:13 WARN  retry, retrying in 1 seconds...
11:00:15 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/17889345/binaries
11:00:15 INFO  nlines 0
11:00:15 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
11:00:15 INFO  
11:00:15 INFO  cleanup:
11:00:15 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
11:00:16 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
11:00:16 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
11:00:21 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
11:00:21 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
11:00:21 INFO  
11:00:21 INFO  Test duration: 32.55 secs
11:00:21 INFO  Test final outcome:  FAILED
11:00:21 INFO  Test outcome reason: Assert that (True == value) with value=False
11:00:21 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>11:14:50 INFO  ==============================================================
11:14:50 INFO  Id:    mapper_awaits_before_reconnect
11:14:50 INFO  Dir:   tedge_mapper_c8y/
11:14:50 INFO  Title: Validate mapper doesn't reconnect too often
11:14:50 INFO  ==============================================================
11:14:50 INFO  Started mosquitto_sub with process id 19449
11:14:50 INFO  Waiting for 0.1 seconds
11:14:51 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
11:14:51 INFO  Waiting for 5.0 seconds
11:14:56 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
11:14:57 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
11:14:57 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
11:14:57 INFO  Line count on file tedge_mapper_journal.out ... passed
11:14:58 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
11:14:58 WARN  Aborted test due to FAILED outcome
11:14:58 INFO  
11:14:58 INFO  Test duration: 7.38 secs
11:14:58 INFO  Test final outcome:  FAILED
11:14:58 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
11:14:58 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>11:01:51 INFO  ==============================================================
11:01:51 INFO  Id:    monitoring_smaller_interval
11:01:51 INFO  Dir:   monitoring/
11:01:51 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
11:01:51 INFO  ==============================================================
11:01:52 INFO  Executed sudo&lt;collectd&gt;, exit status 0
11:01:52 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:01:52 INFO  Started sudo&lt;tedge_sub&gt; with process id 15593
11:01:53 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:01:53 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:01:54 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:01:55 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:01:55 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:01:56 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:01:56 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:01:57 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:01:58 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:01:58 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:01:59 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:01:59 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:02:00 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:02:00 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:02:01 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:02:01 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:02:02 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:02:03 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:02:03 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:02:04 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:02:05 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
11:02:05 INFO  0

11:02:05 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
11:02:05 WARN  TypeError: 'int' object is not subscriptable ... blocked 
11:02:05 INFO  
11:02:05 INFO  cleanup:
11:02:05 INFO  monitoring_cleanup
11:02:06 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:02:06 INFO  
11:02:06 INFO  Test duration: 15.00 secs
11:02:06 INFO  Test final outcome:  BLOCKED
11:02:06 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
11:02:06 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>11:02:06 INFO  ==============================================================
11:02:06 INFO  Id:    monitoring_with_collectd
11:02:06 INFO  Dir:   monitoring/
11:02:06 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
11:02:06 INFO  ==============================================================
11:02:06 INFO  Test execution disabled due to #792 (instable) ... skipped
11:02:06 INFO  
11:02:06 INFO  Test duration: 0.02 secs
11:02:06 INFO  Test final outcome:  SKIPPED
11:02:06 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
11:02:06 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>11:02:06 INFO  ==============================================================
11:02:06 INFO  Id:    monitoring_with_simulated_messages
11:02:06 INFO  Dir:   monitoring/
11:02:06 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
11:02:06 INFO  ==============================================================
11:02:07 INFO  Executed sudo&lt;collectd&gt;, exit status 0
11:02:07 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:02:07 INFO  Started sudo&lt;tedge_sub&gt; with process id 15646
11:02:08 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
11:02:08 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
11:02:10 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
11:02:10 INFO  0

11:02:10 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
11:02:10 WARN  TypeError: 'int' object is not subscriptable ... blocked 
11:02:10 INFO  
11:02:10 INFO  cleanup:
11:02:10 INFO  monitoring_cleanup
11:02:11 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
11:02:11 INFO  
11:02:11 INFO  Test duration: 4.44 secs
11:02:11 INFO  Test final outcome:  BLOCKED
11:02:11 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
11:02:11 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>11:00:56 INFO  ==============================================================
11:00:56 INFO  Id:    mqtt_port_change_connection_works
11:00:56 INFO  Dir:   misc_features/
11:00:56 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
11:00:56 INFO  ==============================================================
11:00:56 INFO  failing intermitently ... skipped
11:00:56 INFO  
11:00:56 INFO  Test duration: 0.02 secs
11:00:56 INFO  Test final outcome:  SKIPPED
11:00:56 INFO  Test outcome reason: failing intermitently
11:00:56 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers">
		<failure message="FAILED: Grep on tedge_mapper2.err contains &quot;Error: Couldn't acquire file lock.&quot;" type="FAILED" />
		<system-out>11:14:58 INFO  ==============================================================
11:14:58 INFO  Id:    runtime_start_two_mappers
11:14:58 INFO  Dir:   tedge_mapper_c8y/
11:14:58 INFO  Title: Validate two mappers can't be started at the same time
11:14:58 INFO  ==============================================================
11:14:58 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
11:14:58 INFO  Waiting for 0.1 seconds
11:14:59 INFO  Executed sudo&lt;tedge_mapper2&gt;, exit status 1
11:14:59 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
11:14:59 WARN  Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock." ... failed [run.py:46]
11:14:59 INFO  
11:14:59 INFO  Test duration: 1.74 secs
11:14:59 INFO  Test final outcome:  FAILED
11:14:59 INFO  Test outcome reason: Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock."
11:14:59 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>11:12:14 INFO  ==============================================================
11:12:14 INFO  Id:    tedge_agent_user_sudo_access
11:12:14 INFO  Dir:   tedge/
11:12:14 INFO  Title: Validate tedge-agent user has a limited sudo right
11:12:14 INFO  ==============================================================
11:12:14 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
11:12:14 INFO  
11:12:14 INFO  Test duration: 0.02 secs
11:12:14 INFO  Test final outcome:  SKIPPED
11:12:14 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
11:12:14 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>11:16:34 INFO  ==============================================================
11:16:34 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
11:16:34 INFO  Dir:   tedge_mapper_c8y/
11:16:34 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
11:16:34 INFO  ==============================================================
11:16:35 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
11:16:35 INFO  Started sudo&lt;tedge_sub&gt; with process id 20040
11:16:35 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
11:16:36 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
11:16:36 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
11:16:36 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
11:16:36 INFO  
11:16:36 INFO  cleanup:
11:16:36 INFO  mapper_cleanup
11:16:37 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
11:16:37 INFO  
11:16:37 INFO  Test duration: 2.95 secs
11:16:37 INFO  Test final outcome:  FAILED
11:16:37 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
11:16:37 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>